temporal/t.rast.univar/t.rast.univar.py

```diff
@@ -87,11 +93,24 @@ def main():
     if output == "-":
         output = None
 
+    # Check if zones map exists and is of type CELL
+    if zones:
+        if gs.raster.raster_info(zones)["datatype"] != "CELL":
+            gs.fatal(_("Zoning raster must be of type CELL"))
+
     tgis.print_gridded_dataset_univar_statistics(
-        "strds", input, output, where, extended, no_header, separator, rast_region
+        "strds",
+        input,
+        output,
+        where,
+        extended,
+        no_header,
+        separator,
+        rast_region,
+        zones,
     )
 
 
 if __name__ == "__main__":
-    options, flags = grass.parser()
+    options, flags = gs.parser()
```

Comment:
Since you are changing it anyway, can you move this to main to avoid an unnecessary global variable.

Reply:
Adressed in [5ba0978](https://github.com/OSGeo/grass/pull/2588/commits/5ba09782ef9c3e7df0c9d0613e51a8624ada1b42) and [086b431](https://github.com/OSGeo/grass/pull/2588/commits/086b4317bf07f921035fd45bd4c515b0d257ed9f)